app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  extend FriendlyId
+  friendly_id :name, use: :slugged
+  belongs_to :user
+
+  def name
+    data = ''
+    description.each_char { |i| data.size <= 10 ? data << i : break }
+    data
+  end
```

Comment:
I am unable to create a post, I get the below error. 
![review_fb_abu](https://user-images.githubusercontent.com/22497270/71301473-c6c82880-239f-11ea-8b7d-e2e14e4eba45.png)